doc/rados/operations/pools.rst

```diff
@@ -167,17 +181,20 @@ following:
 .. describe:: [crush-rule-name]
 
    The name of the CRUSH rule to use for this pool. The specified rule must
-   exist; otherwise the command will fail.
+   already exist, otherwise the command will fail.
 
    :Type: String
    :Required: No.
-   :Default: For **replicated** pools, it is the rule specified by the :confval:`osd_pool_default_crush_rule` configuration variable. This rule must exist.  For **erasure** pools, it is the ``erasure-code`` rule if the ``default`` `erasure code profile`_ is used or the ``{pool-name}`` rule  if not. This rule will be created implicitly if it doesn't already exist.
+   :Default: For ``replicated`` pools, it is by default the rule specified by the :confval:`osd_pool_default_crush_rule` configuration option. This rule must exist.  For ``erasure`` pools, it is the ``erasure-code`` rule if the ``default`` `erasure code profile`_ is used or the ``{pool-name}`` rule  if not. This rule will be created implicitly if it doesn't already exist.
 
 .. describe:: [erasure-code-profile=profile]
 
-   For **erasure** pools only. Instructs Ceph to use the specified `erasure
-   code profile`_. This profile must be an existing profile as defined by **osd
-   erasure-code-profile set**.
+   For ``erasure`` pools only. Instructs Ceph to use the specified `erasure
+   code profile`_. This profile must be an existing profile as defined via
+   the dashboard or invoking ``osd erasure-code-profile set``.  Note that
+   changing the EC profile of a pool after creation does *not* take effect.
```

Comment:
s/changing/changes to/

s/does *not*/do *not*/

Reply:
Pushing